cypress/e2e/aboutpage.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe("About page tests", () => {
+  beforeEach(() => {
+    cy.visit("/about",  { timeout: 30000 });
+  });
+
+  it("About header renders", () => {
+    cy.get('[data-cy="about-header-header"]')
+      .should("be.visible")
+      .and("contain", "Transnational Job Listing Channel");
+    cy.get('[data-cy="about-header-body"]')
+      .should("be.visible")
+      .and("contain", "So many jobs available, all you have to do is keep up with  our posts. Check below for recent job openings.");
```

Comment:
We could refactor this to be a little shorter using `contains` like I suggest below, super optional.